.github/actions/setup/action.yaml

```
@@ -69,3 +69,11 @@ runs:
       run: |
         echo "MARU_AUTH=\"{\"raw.githubusercontent.com\": \"${{ inputs.ghToken }}\"}\"" >> "GITHUB_ENV"
       shell: bash
+
+    - name: Rapidfort Login
+      shell: bash
+      env:
+        RF_ACCESS_ID: ${{ inputs.rfAccessId }}
+        RF_SECRET_ACCESS_KEY: ${{ inputs.rfSecretAccessKey }}
+        RF_ROOT_URL: ${{ inputs.rfRootUrl }}
+      run: echo "$RF_SECRET_ACCESS_KEY" | docker login "$RF_ROOT_URL" -u "$RF_ACCESS_ID" --password-stdin
```

Definitely a nit but we can use zarf tools registry here like we do for IB/GHCR.
```suggestion
      run: echo "${{ env.RF_SECRET_ACCESS_KEY }}" | uds zarf tools registry login -u "${{ env.RF_ACCESS_ID }}" --password-stdin "${{ env.RF_ROOT_URL }}"
```